src/components/styles/Trip.css

```diff
@@ -1,5 +1,117 @@
+.tripDetailsHeader {
+  padding-top: 20px;
+  padding-left: 16px;
+  display: flex;
+  justify-self: left;
+  justify-content: left;
+  align-items: center;
+  margin-left: 20px;
+  z-index: 1;
+}
+
 .TripDetails {
   display: flex;
   flex-direction: column;
+}
+
+.TripDetails h2 {
+  font-size: 1.9em;
+  margin-left: -115px;
+  margin-bottom: 10px;
+  margin-top: 5px;
+  background-color: rgba(37, 57, 73, 0.95);
+  color: white;
+  width: 300px;
+  text-align: end;
+  border-radius: 50px;
+  padding-right: 20px;
+  border: solid 2px rgb(255, 255, 255);
+  box-shadow: 2px 2px 5px #181212;
+}
+
+.tripDetailsContent {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  padding: 10px;
+  color: white;
+  align-self: center;
+  border-radius: 10px;
+  margin-top: 15px;
   align-items: center;
```

Comment:
I think I would prefer if you'd align the items left. It looks a bit busy
<img width="339" alt="Screenshot 2021-05-07 at 09 15 12" src="https://user-images.githubusercontent.com/79857938/117412297-e9b39800-af14-11eb-9856-fb5b5baa29b6.png">